test/fixtures/templates/preserve-ie-conditional-comments/test.js

```diff
@@ -0,0 +1,3 @@
+exports.templateData = {
```

Comment:
The testing harness for Marko scans all of the directories in `test/fixtures/templates` and automatically runs all of the tests. See: https://github.com/raptorjs/marko/blob/209a4de099e855ad86e73ec2383a25bea0d766d7/test/render-test.js

To run all of the tests:

```
npm test
```

To test a specific template:

```
TEST=preserve-ie-conditional-comments npm run test-fast
```